runtime-config-linux.md

```diff
@@ -201,6 +201,8 @@ The following parameters can be specified to setup the controller:
 
 * **`kernel`** *(uint64, optional)* - sets hard limit for kernel memory
 
+* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
```

Comment:
i didn't address this comment

On Mon, Apr 11, 2016 at 6:59 PM, W. Trevor King <notifications@github.com>
wrote:

> In runtime-config-linux.md
> <https://github.com/opencontainers/runtime-spec/pull/235#discussion_r59296300>
> :
>
> > @@ -201,6 +201,8 @@ The following parameters can be specified to setup the controller:
> >
> >  * **`kernel`** *(uint64, optional)* - sets hard limit for kernel memory
> >
> > +* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
>
> On Mon, Apr 11, 2016 at 03:54:09PM -0700, Vish Kannan wrote: > +*
> **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in
> tcp using Oops.. @wking <https://github.com/wking> Can you post a PR to
> fix this ?
> @vbatts <https://github.com/vbatts> just pushed c1612af
> <https://github.com/opencontainers/runtime-spec/commit/c1612af9b80dd828f8fccea3bfca5300281d37b9>
> → d4e7326
> <https://github.com/opencontainers/runtime-spec/commit/d4e7326d50217fc01b0f2bc5acfcf764dd604d85>
> in #370 <https://github.com/opencontainers/runtime-spec/pull/370> which
> should cover this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/235/files/e9a6d94848a3be8a65bc68c3b2c52639041504ed#r59296300>
>